acceptance/creator_test.go

```diff
@@ -260,9 +264,16 @@ func testCreatorFunc(platformAPI string) func(t *testing.T, when spec.G, it spec
 								"--env", "CNB_REGISTRY_AUTH="+createRegAuthConfig,
 								"--network", createRegNetwork,
 								"--volume", dirCache+":/cache",
+								"--volume", dirLaunchCache+":"+ctrPath("/launch-cache"),
 							)...),
 							h.WithArgs(createArgs...),
 						)
+						// check that launch cache was used
+						duration := time.Now().Sub(startTime)
+						t.Logf("Build duration: %s", duration)
+						if duration > 3*time.Second {
+							t.Fatalf("Expected second build to complete in less than 3 seconds; took %s", duration)
+						}
```

Comment:
I'm a bit wary of this test, as it might introduce flakes. But this seemed like the most straightforward way to test that the change does what we want. Open to suggestions...

Reply:
Yeah, it could become a problem but I'm OK with it as a general idea to make sure there are no large regressions.